libs/execution_context/ebpf_native.c

```diff
@@ -1398,3 +1452,42 @@ ebpf_native_get_count_of_maps(_In_ const GUID* module_id, _Out_ size_t* count_of
     ebpf_lock_unlock(&_ebpf_native_client_table_lock, state);
     EBPF_RETURN_RESULT(result);
 }
+
+static ebpf_result_t
+_ebpf_native_helper_address_changed(_Inout_ ebpf_program_t* program, _Inout_opt_ void* context)
```

Comment:
It looks like the callback context will _always_ be of ```ebpf_native_helper_address_changed_context_t``` type, so we might want to make the ```context``` pointer parameter to be of that concrete type too.

Reply:
It has to be of type ebpf_helper_function_addresses_changed_callback_t, which accepts void as this is called for all program types.